### PR TITLE
AB#88303 added question 5 page

### PR DESCRIPTION
**Pull request by rafalgajewski**

### CODE REVIEW

File: src/HE.Investment.AHP.Domain/Site/CommandHandlers/ProvideSection106CapitalFundingEligibilityCommandHandler.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        : base(siteRepository, accountUserContext, logger)
+    {
+    }
+
+    public Task<OperationResult> Handle(ProvideSection106CapitalFundingEligibilityCommand request, CancellationToken cancellationToken)
+    {
+        return Perform(
+            site =>
+            {
+                var currentSection106 = site.Section106 ?? new Section106();
```
Comment (Carq) [inline src/HE.Investment.AHP.Domain/Site/CommandHandlers/ProvideSection106CapitalFundingEligibilityCommandHandler.cs:25]: Sections106 always exists so this check is redundant.

**Review by Carq — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/Workflows/SiteWorkflow.cs [hunk 2]
```diff
@@ -59,5 +59,13 @@ private void ConfigureTransitions()
         _machine.Configure(SiteWorkflowState.Section106AdditionalAffordableHousing)
             .Permit(Trigger.Continue, SiteWorkflowState.Section106CapitalFundingEligibility)
             .Permit(Trigger.Back, SiteWorkflowState.Section106AffordableHousing);
+
+        _machine.Configure(SiteWorkflowState.Section106CapitalFundingEligibility)
+            .PermitIf(Trigger.Continue, SiteWorkflowState.Ineligable, () => _siteModel?.Section106CapitalFundingEligibility == true)
```
Comment (jjankowski87) [inline src/HE.Investment.AHP.WWW/Workflows/SiteWorkflow.cs:64]: Ineligable -> Ineligible

**Review by jjankowski87 — APPROVED** [review]

**Review by Antek012 — APPROVED** [review]

File: src/HE.Investment.AHP.Domain/Site/CommandHandlers/ProvideSection106CapitalFundingEligibilityCommandHandler.cs (hunk 1 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Domain/Site/CommandHandlers/ProvideSection106CapitalFundingEligibilityCommandHandler.cs]: agree

File: src/HE.Investment.AHP.WWW/Workflows/SiteWorkflow.cs (hunk 2 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Workflows/SiteWorkflow.cs]: fixed

**Pull request merged**